Website/web.config

```diff
@@ -159,7 +159,7 @@
       </expressionBuilders>
     </compilation>
     <!-- permits errors to be displayed for remote clients -->
-    <customErrors mode="RemoteOnly"/>
+    <customErrors mode="off"/>
```

Comment:
Do not commit your web.config changes unless they need to be applied to all developers.  :)